specification/hdinsight/resource-manager/Microsoft.HDInsight/stable/2018-06-01-preview/applications.json

```diff
@@ -250,6 +250,14 @@
                     "type": "integer",
                     "format": "int32",
                     "description": "The public port to connect to."
+                },
+                "subDomainSuffix":{
+                    "type": "string",
+                    "description": "The subDomainSuffix of the application and can not greater than 3 characters."
```

Comment:
If I understand correctly, the description should say "The subDomainSuffix of the application.  It cannot be greater than 3 characters."